Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -637,10 +637,17 @@ private extension VitalInfo {
 
     func asJsRefreshRate() -> RUMViewEvent.View.JsRefreshRate {
         return RUMViewEvent.View.JsRefreshRate(
-            average: meanValue ?? 0.0,
-            max: maxValue ?? 0.0,
+            average: invertValue(value: meanValue) * 1_000,
+            max: invertValue(value: minValue) * 1_000,
             metricMax: 60.0,
-            min: minValue ?? 0.0
+            min: invertValue(value: maxValue) * 1_000
         )
     }
 }
+
+internal func invertValue(value: Double?) -> Double {
+    if value == 0.0 || value == nil {
+        return 0.0
+    }
+    return 1 / (value ?? 1)
+}
```

Comment:
This can be moved to a `var` in our extension in https://github.com/DataDog/dd-sdk-ios/blob/develop/Sources/Datadog/Utils/SwiftExtensions.swift#L19L26

it can be:
```
var inverted: Double {
    return self == 0 ? 0 : 1 / self
}
```